bot.py

```diff
@@ -298,23 +322,15 @@ def translate_slack_usernames(self, unchecked_authors):
                 )
             else:
                 await self.say(channel_id, "Oooopps! Invalid command format")
-        except BotException as ex:
+        except (InputException, ReleaseException) as ex:
             log.exception("A BotException was raised:")
             await self.say(channel_id, "Oops, something went wrong: {}".format(ex))
         except:  # pylint: disable=bare-except
             log.exception("Exception found when handling a message")
-            await self.say(channel_id, "Oops, something went wrong...")
-
-
-class BotException(Exception):
-    """Exception raised for invalid input.
-
-    Args:
-        message (str): explanation of the error
-    """
-    def __init__(self, message):
-        super().__init__(message)
-        self.message = message
+            await self.say(
+                channel_id,
+                "No! Perry the Platypus, don't do it! Don't push the self-destruct button. This one right here.",
```

Comment:
Should we include some indication that this indicates an unhandled exception? If I saw this message in slack I would have no idea what it's trying to tell me

Reply:
I'm not sure, I feel like it would ruin the joke to explain it. I like having it just spontaneously appear